content/help/sso/jsconnect/index.md

```diff
@@ -81,11 +86,186 @@ You implement site-wide SSO when Vanilla is as a full site. With this method you
 ### Method 2: Embedded SSO
 Vanilla also has the ability to be embedded in an html page either as an entire site or just as embedded comments. With this scenario you need to create a special SSO string that can be passed to the embedded Vanilla and provide login information.
 
+## Technical Overview
+
+For jsConnect to work you'll put some code on your site that identifies your users in a way that Vanilla can understand. Vanilla then requests this information when the user wants to sign in and synchronizes the user with Vanilla. This is done via [JSONP](http://en.wikipedia.org/wiki/JSONP), which allows us to send your cookies from your site to you and obtain login information.
+
+Vanilla's web-based single sign on (SSO) has 2 parts:
+
+1. Your SSO endpoint (based on one of our example libraries) on _your_ site.
+2. Vanilla's jsConnect plugin.
+
+jsConnect pings your endpoint whenever it wants to know if a user is logged in on your site. It does this using the user's current session, so you can use your normal "is logged in" detection to determine this - nothing fancy here.
+
+### Your Endpoint & Client Libraries
+
+Your endpoint needs to say 1 of 3 things when it's called:
+
+1. The current user is a **guest** (not logged in).
+2. User is logged in here, but this wasn't a secure request (no signature + timestamp), so here's their **name & photo only**.
+3. User is logged in AND this was securely requested, so here's **all the user data** (including email and a unique ID from your system, which are required).
+
+Our example libraries help you structure the output so that jsConnect can read it correctly. **If we provide a client library that you can use then we highly recommend using it.**
+
+These libraries are open source and maintained on Github:
+
+- [PHP jsConnect client library](https://github.com/vanillaforums/jsConnectPHP)
+- [Ruby jsConnect client library](https://github.com/vanillaforums/jsConnectRuby)
+- [Java jsConnect client library](https://github.com/vanillaforums/jsConnectJava)
+- [.NET jsConnect client library](https://github.com/vanillaforums/jsConnectdotNet)
+- [Python jsConnect client library](https://github.com/vanilla/python-jsconnect)
+
+
+All of these libraries have one file with all of the library code you'll need and one file that gives an example usage. They also have a readme that tells you which file is which. _Please note that these libraries don't support [embedded SSO](/help/features/sso/jsconnect/embed)._
+
+If your site is programmed in a language that doesn't have a client library then we provide documentation on our jsConnect protocol. Have a look at the [jsConnect technical implementation guide](/help/sso/jsconnect-quickstart).
```

Comment:
Go over all the links for pages you've combined. For example, the links to "/help/sso/jsconnect-quickstart" are now redirecting to the same page, not very useful. You'll want to replace these with an anchor.